app/config.js

```diff
@@ -2,6 +2,9 @@ module.exports = {
   // App name
   appName: 'NHSD Buying Catalogue Admin',
 
+  // Base URL
+  baseUrl: 'admin',
```

Comment:
does this base url have anything to do with the urls in production? everyting in this repo is on the /organisations url

Reply:
Initially Stuart had configured ingress to point ```{server_url}/admin``` to the route (```/```) of the admin app but because of the styling issues we now have to match the structure locally so that ```{server_url}/admin``` points to ```/admin``` in the admin app so we can have the styles and links working both locally and with ingress.

Reply:
ah ok so this is nginx doing the redirect so is the redirect I commented on so that it will work locally?

Reply:
yeah

Reply:
We're gonna see if we can get it working for admin then will apply it to all the other FEs